intro_to_rl.ipynb

```diff
@@ -0,0 +1,3287 @@
+{
```

Comment:
<p>To avoid confusion, we should probably either explain <code>evaluation</code> or remove this extra complexity. Not sure if it explained below.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>removed</p>